mne/commands/tests/test_commands.py

```diff
@@ -276,6 +277,33 @@ def test_flash_bem(tmpdir):
         assert len(tris) == 5120
 
 
+@pytest.mark.slowtest
+@sample.requires_sample_data
+def test_setup_source_space(tmpdir):
+    """Test mne setup_source_space."""
+    check_usage(mne_setup_source_space, force_help=True)
+    # Using the sample dataset
+    subjects_dir = op.join(sample.data_path(download=False), 'subjects')
+    use_fname = op.join(tmpdir, "sources-src.fif")
+    # Test  command
+    with ArgvSetter(('--src', use_fname, '-d', subjects_dir,
+                     '-s', 'sample', '--morph', 'sample')):
+        mne_setup_source_space.run()
+    src = read_source_spaces(use_fname)
+    assert len(src) == 2
+    with pytest.raises(Exception):
+        with ArgvSetter(('--src', use_fname, '-d', subjects_dir,
+                         '-s', 'sample', '--ico', '3', '--oct', '3')):
+            assert mne_setup_source_space.run()
```

Comment:
the next 8 lines are not executed by the test as the exception is raised as this line.
you need a `with pytest.raises(Exception):` for the 3 cases.

Reply:
Done. Should I cover more test cases as code coverage is decreased ? Other comands seems to only test 1 working case.